site/_guides/tls.md

```diff
@@ -0,0 +1,18 @@
+---
+title: TLS support
+layout: page
+---
+
+# TLS support
+
+Contour 0.3 adds support for HTTPS (TLS/SSL) ingress by integrating Envoy's SNI support.
```

Comment:
Isn't 0.3 ancient history? How about we just say that Contour supports TLS for Ingress and HTTPProxy?